tests/PySys/apt-plugin/apt-install/run.py

```diff
@@ -0,0 +1,49 @@
+import pysys
+from pysys.basetest import BaseTest
+
+"""
+Validate apt plugin install
+
+Using `rolldice` as a guinea pig: [small and without impacts](https://askubuntu.com/questions/422362/very-small-package-for-apt-get-experimentation)
+"""
+
+class AptPluginTest(BaseTest):
```

Comment:
It's a good general pysys practice to have different names for different test classes, which are usually named after the test case. So this class could be named `AptPluginInstallTest` and the other tests could be `AptPluginRemoveTest` and `AptPluginListTest` respectively.

The benefit is when someone searches for these tests from an IDE where they can specifically search for a given class name. We can't rely on file name based search because all test files are named `run.py`. So class name based search is the only way. If you change the class name here, you'll have to update that class name in this test's `pysystest.xml` as well

Reply:
Renamed